bin/osg-notify

```diff
@@ -71,14 +77,14 @@ def generateFullMessage(subject, to, from_name, message, sign=True, keyid=None):
         msg.attach(signmsg)
         msg['Subject'] = subject
         msg['Bcc'] = ", ".join(to)
-        msg['To'] = '"Open Science Grid Helpdesk" <help@opensciencegrid.org>'
-        msg['From'] = '"%s" <help@opensciencegrid.org>' % from_name
+        msg['To'] = '"%s" <%s>' % (to_name, from_addr)
```

Comment:
```suggestion
        msg['To'] = '"%s" <%s>' % (to_name, to_addr)
```
and then add `to_addr = from_addr` earlier in the code. I know we're reusing the address but anybody looking at it would think it's a bug...